frontend/src/pages/components/displayImages.js

```diff
@@ -284,6 +277,8 @@ function PageCounter() {
  * 
  * Col 2:
  *  | Comments
+ * 
+ * props: cards
```

Comment:
line 287 (it wouldn't let me put a comment there) should be card.userId._id instead of props.card.userId._id. I thought this was changed in main before, but it could have been lost in the merge. It caused an issue with the single image view.

Reply:
Thanks for the catch!